app/models/aytq_feedback.rb

```diff
@@ -0,0 +1,14 @@
+class AYTQFeedback < ApplicationRecord
```

Comment:
The Trello card suggests using the same model/table as Check. I think it's fine to just add a `service` enum to `Feedback` with values `aytq` and `check`.